src/content/peerconnection/upgrade/js/test.js

```diff
@@ -15,7 +15,7 @@ let driver;
 const path = '/src/content/peerconnection/upgrade/index.html';
 const url = `${process.env.BASEURL ? process.env.BASEURL : ('file://' + process.cwd())}${path}`;
 
-describe('peerconnection upgrade from audio-only to audio-video', () => {
+xdescribe('peerconnection upgrade from audio-only to audio-video', () => {
```

Comment:
I take it x-prefixing is the accepted idiom for "disable"? Or does this need a comment?

Reply:
yes. I should try describe.skip which is more readable